domain/model/service/modelservice.go

```diff
@@ -30,6 +30,9 @@ type ModelState interface {
 	// GetModelMetrics returns the model metrics information set in the
 	// database.
 	GetModelMetrics(context.Context) (coremodel.ModelMetrics, error)
+
+	// GetModelCloudType returns the read only model cloud type set in the database.
```

Comment:
We can safely remove the read only portion of this. Just the model cloud type is fine.